CUETools.Codecs.FlaCuda/FlaCudaWriter.cs

```diff
@@ -1574,7 +1574,7 @@ public unsafe void InitTasks()
 				//cuda.LoadModule(System.IO.Path.Combine(Environment.CurrentDirectory, "flacuda.cubin"));
 				if (_IO == null)
 					_IO = new FileStream(_path, FileMode.Create, FileAccess.Write, FileShare.Read);
-				int header_size = flake_encode_init();
+				int header_size = Flake_encode_init();

```

Comment:
Please undo. The lower case letter at the beginning of the method name should stay, like in other methods.